pingcap/tidb
# server: add GRPC debug log when set a Env var

### What problem does this PR solve? 

current pingcap/log will override grpclog, we cannot see grpc log even if set

```
GRPC_GO_LOG_VERBOSITY_LEVEL=99 GRPC_GO_LOG_SEVERITY_LEVEL=info
```


### What is changed and how it works?

reset grpclog if TiDB have `GRPC_DEBUG` ENV var

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test 

```
run tidb with 

GRPC_DEBUG=1 ./tidb-server --store tikv -path 127.0.0.1:2379

and see stderr.log


INFO: 2019/12/19 15:50:24 parsed scheme: ""
INFO: 2019/12/19 15:50:24 scheme "" not registered, fallback to default scheme
INFO: 2019/12/19 15:50:24 ccResolverWrapper: sending update to cc: {[{127.0.0.1:40500 0  <nil>}] <nil>}
INFO: 2019/12/19 15:50:24 ClientConn switching balancer to "pick_first"
INFO: 2019/12/19 15:50:24 parsed scheme: ""
INFO: 2019/12/19 15:50:24 scheme "" not registered, fallback to default scheme
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a4410, CONNECTING
INFO: 2019/12/19 15:50:24 ccResolverWrapper: sending update to cc: {[{127.0.0.1:40500 0  <nil>}] <nil>}
INFO: 2019/12/19 15:50:24 ClientConn switching balancer to "pick_first"
INFO: 2019/12/19 15:50:24 parsed scheme: ""
INFO: 2019/12/19 15:50:24 scheme "" not registered, fallback to default scheme
INFO: 2019/12/19 15:50:24 ccResolverWrapper: sending update to cc: {[{127.0.0.1:40500 0  <nil>}] <nil>}
INFO: 2019/12/19 15:50:24 ClientConn switching balancer to "pick_first"
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a50e0, CONNECTING
INFO: 2019/12/19 15:50:24 parsed scheme: ""
INFO: 2019/12/19 15:50:24 scheme "" not registered, fallback to default scheme
INFO: 2019/12/19 15:50:24 ccResolverWrapper: sending update to cc: {[{127.0.0.1:40500 0  <nil>}] <nil>}
INFO: 2019/12/19 15:50:24 ClientConn switching balancer to "pick_first"
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a54c0, CONNECTING
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a53a0, CONNECTING
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a54c0, READY
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a53a0, READY
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a50e0, READY
INFO: 2019/12/19 15:50:24 pickfirstBalancer: HandleSubConnStateChange: 0xc0001a4410, READY

```

Code changes

 - impl

Side effects

 - N/A

Related changes

 - Need to cherry-pick to the release branch

Release note

 - add GRPC debug log when set a Env var.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pingcap/tidb/14140)
